app/interactors/send_timer_reminders.rb

```diff
@@ -0,0 +1,18 @@
+class SendTimerReminders
+  include Interactor
+  include HasHarvest
+
+  before do
+    context.date ||= Date.current
+  end
+
+  def call
+    User.all.each do |user|
+      next if user.timer_reminder_sent_on.try(:today?)
```

Comment:
I'm going to go the route of assuming that the user already has a day recorded because of how often new hours are fetched for the day and because of how this particular interactor is scheduled for later in the morning. That also gives us the benefit of not having to call out to Harvest so we can keep this all in the model.